packages/Breadcrumbs/stories/examples/AllVariations.js

```diff
@@ -83,6 +83,42 @@ const AllVariations = () => {
       <Breadcrumbs>
         <Breadcrumbs.Link href={URL}>Breadcrumb 1</Breadcrumbs.Link>
       </Breadcrumbs>
+
+      <Gap />
+
+      <Heading level={3} displayLevel={5} isLight>
+        Not clickable breadcrumb item
+      </Heading>
+
+      <Breadcrumbs>
+        <Breadcrumbs.Link
+          as="span"
+          style={{
+            fontWeight: "bold",
+            color: "#3f3d3c",
+          }}
+        >
+          Breadcrumb 1 with long content, Breadcrumb 1 with long content.
```

Comment:
I think:
- the text should just be `Breadcrumb 1` and `Breadcrumb 4` (not long text)
- get rid of the `style`
 
Demonstrating multiple features in a single story just complicates things for the consumer.

Reply:
Sure